kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/MobiComAttachmentSelectorActivity.java

```diff
@@ -183,64 +156,55 @@ public void onClick(View v) {
                             startActivity(startConversationActivity);
                             finish();
                         } catch (Exception e) {
+                            Toast.makeText(MobiComAttachmentSelectorActivity.this, e.getMessage(), Toast.LENGTH_SHORT).show();
                             e.printStackTrace();
                         }
                     }
-
                 } else {
                     Intent intent = new Intent();
                     intent.putParcelableArrayListExtra(MULTISELECT_SELECTED_FILES, attachmentFileList);
                     intent.putExtra(MULTISELECT_MESSAGE, messageEditText.getText().toString());
                     setResult(RESULT_OK, intent);
                     finish();
                 }
+            }
+        });
 
+        //what to do before and after the doInBackground of the FileTaskAsync
+        prePostUIMethodsFileAsyncTask = new PrePostUIMethods() {
+            ProgressDialog progressDialog;
 
+            @Override
+            public void preTaskUIMethod() {
+                progressDialog = ProgressDialog.show(MobiComAttachmentSelectorActivity.this, "Wait",
```

Comment:
@shubhamtewari this "Wait" will be visible to users? if yes, we can move it to strings.xml

Reply:
moved